### PR TITLE
MMCA-4894 | Landing page notification for Requested Cash Account Transactions

### PR DESCRIPTION
**Pull request by Neozxc**
Task:
- On login, check mongodb for cash account statement notification in the notification-store collection
- Display the notification on the landing page

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -167,6 +167,7 @@ cf.customs-financials-home.notification.requested-duty-deferment=Your requested
 cf.customs-financials-home.notification.requested-adjustments=Your requested import adjustments statements are available to view
 cf.customs-financials-home.notification.requested-postponed-vat=Your requested postponed import VAT statements are available to view
 cf.customs-financials-home.notification.authorities=Your requested list of accounts you have authority to use is available
+cf.customs-financials-home.notification.cash-account-statement=Your requested cash account statements are available to view
```
Comment (AdamCoops) [inline conf/messages.en:170]: Does this need a welsh translation?

Comment (Neozxc) [reply conf/messages.en:170]: Welsh translation should be ready by tomorrow. Hold this PR open.

**Comment by AdamCoops** [comment]
looks good

**Review by AdamCoops — DISMISSED** [review]

File: app/domain/SdesFile.scala [hunk 2]
```diff
@@ -317,3 +325,23 @@ case class PostponedVatStatementFileMetadata(periodStartYear: Int,
...
+
+case class CashAccountStatementFile(filename: String,
+                                    downloadURL: String,
+                                    size: Long,
+                                    metadata: CashAccountStatementFileMetadata,
+                                    eori: String)
+                                   (implicit messages: Messages
+                                   ) extends Ordered[CashAccountStatementFile] with SdesFile {
+
+  val formattedSize: String = Formatters.fileSize(size)
```
Comment (tyagikumarvineet) [inline app/domain/SdesFile.scala:337]: Could you please add tests for this val, formattedMonth and for the compare method (if possible) ?

**Review by tyagikumarvineet — APPROVED** [review]

**Pull request merged**